kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue

```diff
@@ -0,0 +1,159 @@
+      audience, in the broadest sense of the word 'accessible'. It should run on
+      wide variety of devices, render correctly on a wide variety of screen sizes
+      and assistive technologies, be understandable to people from a wide variety
+      of backgrounds, and be usable by people with a wide variety of abilities and
+      disabilities.
+    </p>
+
+    <p>
+      At a high level, we follow the web content accessibility guideline that for
+      all users, information should be:
+    </p>
+
+    <ul>
+      <li>Perceivable</li>
+      <li>Operable</li>
+      <li>Understandable</li>
+      <li>Robust</li>
+    </ul>
+
+    <h2>Window size and zoom</h2>
+
+    <p>
+      Users will use devices that have a wide range of screen sizes. Additionally,
+      some users with poor vision may zoom far in, which effectively lowers the
+      size.
+    </p>
+
+    <p>
+      We should aim to allow zooming up to 200%, and down to 480x600 px in both
+      profile and landscape. The app should always remain functional in all extreme
+      screen sizes. Depending on the context, we may want to put additional effort
+      into creating custom layouts optimized for particular screen sizes. For
+      example, we should put additional effort into optimizing the mobile Learner
+      experience, while for administrators this might not be as critical.
+    </p>
+
+    <p>
+      When designing and building layouts, we use the breakpoints defined by
+      Material Design. These are exposed in our app by the
+      <code>responsiveWindow</code> Vue component mixin. We should never use CSS
+      media queries.
+    </p>
+
+    <h2>Keyboard navigation</h2>
+
+    <p>
+      Every interaction that can be performed with a mouse or touchscreen should be
+      accessible with only the keyboard. As much as possible, we should follow best
+      practices including:
+    </p>
+
+    <ul>
+      <li>high-contrast focus indicators</li>
+      <li>intuitive navigation tab order</li>
+      <li>conventional keyboard shortcuts for common input widgets</li>
+    </ul>
+
+    <p>
+      Using the built-in components outlined in this guide is a good way to ensure
+      that your UI will be navigable by keyboard.
+    </p>
+
+    <h2>Screen reader support</h2>
+
+    <p>
+      All content that can be read by sighted users should be readable by screen
+      readers. Additionally, the app should leverage screen-reader-specific
+      functionality – such as labels, semantically correct HTML tags, and
+      visually-hidden text – that associates descriptive text with functionality.
+    </p>
+
+    <p>
+      Like with keyboard navigation, using the built-in components is a good way to
+      ensure compatibility with screen readers. However, it's also critical to test
+      all new functionality. See this
+      <a href="https://github.com/learningequality/ka-lite/wiki/Accessibility-Resources-(Tools)">
```

Comment:
There is an updated list on Kolibri's own Wiki 🙂 
https://github.com/learningequality/kolibri/wiki/Accessibility-Resources-(Tools)